viewing_party/main.py

```diff
@@ -0,0 +1,157 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title and genre and rating:
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie 
+
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    watched = user_data["watched"].append(movie)
+    return user_data
+    
+    
+
+def add_to_watchlist(user_data, movie):
+    watchlist = user_data["watchlist"].append(movie)
+    return user_data
+
+# def watch_movie(user_data, movie_title):
+#     for key, value  in user_data.items():
+#         user_data["watched"][0] = user_data["watchlist"][0]
+#         user_data["watchlist"][0] = []
+#         return user_data
+
+def watch_movie(user_data, movie_title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == movie_title:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
+            
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    total = 0
+    average = 0.0
+    for movie in user_data["watched"]:
+        if user_data["watched"] == 0:
+          average = 0.0
+          return average
+        else:
+          total += movie["rating"]
+          average = (total / len(user_data["watched"]))
```

Comment:
👍 great idea to check if `user_data["watched"] is empty, but think about doing it *before* the for loop next time; otherwise, you are executing a line of code that might not even need to run

on line 38 you've already declared average as 0.0, no need to declare it again on line 41.

```suggestion
    if len(user_data["watched"]) > 0:
       for movie in user_data["watched"]:
          total += movie["rating"]
          average = (total / len(user_data["watched"]))
```